### PR TITLE
Consolidate NetServer config

### PR DESCRIPTION
**Pull request by digitaldiina**
Goes with https://github.com/SuperOfficeDocs/data-access/pull/86

### CODE REVIEW

File: docs/automation/netserver-scripting/configure.md [hunk 1]
```diff
@@ -25,18 +25,7 @@ The configuration options are:
...
-
-```XML
-<Scripting>
-  <add key="EnableScripting" value="False" />
-  <add key="ScriptPath" value="" />
-  <add key="TimeoutLimit" value="20000" />
-  <add key="MaxTimeouts" value="5" />
-</Scripting>
-```
+The *SoAppConfig.exe* utility will use the [Scripting section][1] of the NetServer configuration.
```
Comment (AnthonyYates) [inline docs/automation/netserver-scripting/configure.md:28]:
```suggestion
The *SoAppConfig.exe* utility sets the [Scripting section][1] of the NetServer configuration.
```

Comment (digitaldiina) [reply docs/automation/netserver-scripting/configure.md]: fixed

File: docs/marketing/forms/recaptcha.md [hunk 2]
```diff
@@ -50,11 +53,9 @@ And these lines under the section `<SuperOffice>`:
 </GoogleRecaptcha>
```
Comment (AnthonyYates) [inline docs/marketing/forms/recaptcha.md:53]: Create an issue to add GoogleRecaptcha section to the Configuration article?

File: docs/mobile/pocket-crm/configure.md [hunk 3]
```diff
@@ -20,23 +21,29 @@ so.client: pocket
...
+```xml
+<add key="LogToFile" value="true" />
+```
 
 ## NetServer configuration - GSM formatted phone numbers
 
-When initiating phone calls from Pocket CRM it is important that the numbers are formatted in a way that the phone can use. SuperOffice NetServer has the functionality to return all phone numbers from the SuperOffice database in a clean formatted way, without spaces, commas, and including country prefix, like this: +4712345678. To enable this feature in NetServer make sure that *web.config* contains this element:
+When initiating phone calls from Pocket CRM the numbers must be formatted in a way that the phone can use. SuperOffice NetServer has the functionality to return all phone numbers from the SuperOffice database in a clean formatted way, without spaces, commas, and including country prefix, like this: +4712345678.
+
+To enable this feature in NetServer make sure that *web.config* contains this element:
```
Comment (AnthonyYates) [inline docs/mobile/pocket-crm/configure.md:34]: Where in the web.config?

AllwaysUseGsmPhoneStyle is in [globalization](https://docs.superoffice.com/api/netserver/config/globalization.html)

File: docs/mobile/pocket-crm/deploy.md [hunk 4]
```diff
@@ -88,6 +88,7 @@ You can send a predefined email with this URL by clicking **Pocket CRM settings*
 [4]: https://play.google.com/store/apps/details?id=no.superoffice.pocket.j2me.SuperOffice
 [5]: http://itunes.apple.com/no/app/superoffice/ce/id445355343
 [6]: install.md
+[7]: ../../../../data-access/docs/netserver/config/soformsauthentication.md
```
Comment (AnthonyYates) [inline docs/mobile/pocket-crm/deploy.md:91]: This link is declared, but not used anywhere. Please place it in the NOTE section referencing the web.config file.

File: docs/onsite/debug/web-client.md [hunk 5]
```diff
@@ -2,71 +2,81 @@
...
 
-We do, however, also have some online services configurable in the *web.config* file. If you experience poor performance on the second login, you may turn off *Logging to SuperOffice Research & Product Developement.* Go to Diagnostics and turn off Log to SuperOffice Research & Product Developement.
+SuperOffice Inbox and Pocket CRM use NetServer service to send emails. If you want to troubleshoot email sending or receiving via the NetServer, select **Log e-mail** and in the field next to this option define where the log files should be created.
 
-## NetServer diagnostics
+Logging will create 2 files - one for incoming emails (communication with IMAP server) and one for outgoing emails (communication with SMTP server).
 
-[Turn on/off in web.config][3]
+> [!NOTE]
+> This is not a debug log for MailLink.
```
Comment (AnthonyYates) [inline docs/onsite/debug/web-client.md:75]: If there is a debug log setting for MailLink, link to it from here.

File: docs/onsite/install/index.yml [hunk 6]
```diff
@@ -157,21 +157,21 @@ landingContent:
       links:
```
Comment (AnthonyYates) [inline docs/onsite/install/index.yml:157]: This file should be reference in /docs/onsite/toc.yml

- name: Install Onsite
  href: index.yml
  items:

File: docs/onsite/install/netserver/index.md [hunk 7]
```diff
@@ -1,20 +1,22 @@
...
 keywords:
 so.topic: howto
 so.envir: onsite
 so.client: web
 ---
 
-# Install NetServer / Web services
+# Install NetServer / web services
 
 NetServer is always installed, but for some scenarios, we recommend it is installed as remote web services.
```
Comment (AnthonyYates) [inline docs/onsite/install/netserver/index.md:15]: Could use an elaboration for what scenarios it is recommended to install as remote web services.

Comment (digitaldiina) [reply docs/onsite/install/netserver/index.md:15]: See issue #169

File: docs/onsite/install/win-client/superoffice-config.md [hunk 8]
```diff
@@ -15,20 +16,18 @@ Used for NetServer config info for the Win client and MailLink.
```
Comment (AnthonyYates) [inline docs/onsite/install/win-client/superoffice-config.md:16]: Moving files around (moving authors cheese) is not cool without verbal agreement.

File: docs/onsite/toc.yml [hunk 9]
```diff
@@ -123,44 +123,37 @@ items:
       - name: Move CRM Server
         href: upgrade/to-81/move-crm-server.md
     - name: To version 9
-      href: upgrade/to-9/index.md        
+      href: upgrade/to-9/index.md
   - name: Config
```
Comment (AnthonyYates) [inline docs/onsite/toc.yml:128]: Please eliminate shortened words in TOC. This section contains "Configuration" or "Configuration Settings".  Conversely, avoid more then two words if possible.

File: docs/onsite/upgrade/to-81/using-old-service-domain.md [hunk 10]
```diff
@@ -15,7 +16,7 @@ so.envir: onsite
 
 1. Run the **SuperOffice SM.win** **SetupExe.exe** installer to upgrade the server and the database.
 
-2. Run the **SuperOffice.CustomerService.exe** to upgrade Service software.
+2. Run the **SuperOffice.CustomerService.exe** file to upgrade Service software.
```
Comment (AnthonyYates) [inline docs/onsite/upgrade/to-81/using-old-service-domain.md:19]:
```suggestion
2. Run **SuperOffice.CustomerService.exe** to upgrade Service software.

```

File: docs/service/config/service-config.md [hunk 11]
```diff
@@ -0,0 +1,14 @@
+---
+title: Service config
```
Comment (AnthonyYates) [inline docs/service/config/service-config.md:2]: What's the meaning of this empty page?

File: docs/service/install/index.md [hunk 12]
```diff
@@ -25,13 +26,13 @@ You need to have a working SuperOffice web services installation before you proc
 
 The installer will copy the necessary files to the Service program folder.
 
-1. Run the SuperOffice.CustomerService.exe
+1. Run the SuperOffice.CustomerService.exe file.
```
Comment (AnthonyYates) [inline docs/service/install/index.md:29]:
```suggestion
1. Run SuperOffice.CustomerService.exe to begin.

```

File: docs/service/install/index.md [hunk 13]
```diff
@@ -169,13 +170,14 @@ These applications in turn will perform tasks like importing email, sending emai
 6. To start the service at boot, right-click **ejSchedule**, select **Properties** and make sure **Startup type** is set to **Automatic**.
 7. Click **Start** to start the service
 
-You are now ready to log in and use Service by opening your client browser and go to *[Hostname]/scripts/ticket.fcgi*
+You are now ready to log in and use Service by opening your client browser and going to *[Hostname]/scripts/ticket.fcgi*
 
 ***
 
 [Previous][15] | [Next][14] | [Back to guide][13]
```
Comment (AnthonyYates) [inline docs/service/install/index.md:177]: Do these really go where to relevant places? From here, next navigates to Install Web. Is that the correct order?

File: docs/service/install/manual-steps.md [hunk 14]
```diff
@@ -23,7 +24,7 @@ If the website is created, then you may create the application first, and just p
 
 ![x -screenshot][img1]
 
-Run the SuperOffice.CustomerService.exe to install the software on the server, the default location is *c:\superoffice*.
+Run the SuperOffice.CustomerService.exe file to install the software on the server, the default location is *c:\superoffice*.
```
Comment (AnthonyYates) [inline docs/service/install/manual-steps.md:27]:
```suggestion
Run SuperOffice.CustomerService.exe to install the software on the server, the default location is *c:\superoffice*.

```

File: docs/travel/toc.yml [hunk 15]
```diff
@@ -40,6 +40,12 @@ items:
     href: travel-gateway/activity-types.md
   - name: Troubleshooting
     href: travel-gateway/troubleshooting.md
+  - name: Config
```
Comment (AnthonyYates) [inline docs/travel/toc.yml:43]: Please change name to "Configuration" or "Configuration settings". 

Do not change folder names.

File: docs/travel/travel-gateway/username-ini.md [hunk 16]
```diff
@@ -20,7 +20,9 @@ so.topic: reference
 ![x -screenshot][img2]
 
 > [!NOTE]
-> It adds the associate ID when you ask for other users and will check the database for this associate's current user group.<br>Documents from other users are only copied down to the travel user, not from travel user to central.
+> It adds the associate ID when you ask for other users and will check the database for this associate's current user group.
```
Comment (AnthonyYates) [inline docs/travel/travel-gateway/username-ini.md:23]: Moving someones cheese again... 😨

**Review by AnthonyYates — CHANGES_REQUESTED** [review]
I've made some comments that need changes.

Moving files into other folders without mutual consent is forbidden. This must be discussed with Martin prior to approval.

**Review by SuperOfficeDevNet — CHANGES_REQUESTED** [review]
See my other comments.

**Review by SuperOfficeDevNet — APPROVED** [review]

**Pull request merged**